modules/i18n/src/translations/en-US/portals/actions.ts

```diff
@@ -162,10 +162,14 @@ export const actions: actionsNS = {
                 description: "{{description}}",
                 message: "Error deleting the action."
             },
-            fetch: {
+            fetchById: {
                 description: "{{description}}",
                 message: "Error fetching the action."
             },
+            fetchByType: {
+                description: "{{description}}",
+                message: "Error fetching actions."
```

Comment:
or `retrieving` maybe. fetching is a bit technical for an average user.

```suggestion
                message: "Error getting actions."
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7168/commits/d6e7e3512f0791c828b5e4fa57cd14a5bfaecdd3